test/config_test/example_configs_test_setup.sh

```diff
@@ -5,3 +5,6 @@ set -e
 DIR="$TEST_TMPDIR"/test/config_test
 mkdir -p "$DIR"
 tar -xvf "$TEST_SRCDIR"/envoy/configs/example_configs.tar -C "$DIR"
+
+# find uses full path to prevent using windows find on windows
+/usr/bin/find "$DIR" -type f | grep -c .yaml > "$TEST_TMPDIR"/config-file-count.txt
```

Comment:
Can we use `/usr/bin/env`'s help? Not sure about Windows 😅.

Reply:
not sure how that would work on windows - just waiting to see if the full path fixes the problem...

Reply:
full path fixeds the find issue

a quick search of "windows /usr/bin/env find" doesnt seem to suggest that would work (or otherwise really) - i think probably just using the full path should be safe

Reply:
Yeah using `#!/usr/bin/env bash`, `/c/Windows/System32` shows up on the PATH first still, to get around that we would have to do some PATH ordering magic somewhere